src/Analyser/TypeSpecifierContext.php

```diff
@@ -25,6 +25,10 @@ private function __construct(?int $value)
 
 	private static function create(?int $value): self
 	{
+		if ($value === null) {
+			$value = '';
```

Comment:
This isn't necessary, it will get cast by the array itself.

Reply:
Agreed, it's just to make what's happening explicit. Is this behaviour intentional?

Reply:
Yeah, it's fine 😊